source/tutorials/fmv.rst

```diff
@@ -1,36 +1,46 @@
 .. _fmv:
 
-Function multi-versioning
+Function Multi-Versioning
```

Comment:
Revisions look great. It's so much cleaner and easy to follow. Just a couple fussy things. 
Please revert header to original. We use `Title case` for our [headers](https://docs.01.org/clearlinux/latest/reference/collaboration/structure-formatting.html). Thanks.

Reply:
Stet.